### PR TITLE
Add Rocket Arrow and Night Express (Anime)

### PR DESCRIPTION
**Pull request by TheRazgriz**
Scripts for the cards "Night Express Knight (Anime)" and "Rocket Express Arrow (Anime)"

Night Express - https://yugipedia.com/wiki/Night_Express_Knight_(anime)
Rocket Arrow - https://yugipedia.com/wiki/Rocket_Arrow_Express_(anime)

@larry126 
@ClementLouis 
@pyrQ

### CODE REVIEW

File: unofficial/c511027120.lua [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+function s.mtcon(e,tp,eg,ep,ev,re,r,rp)
+	return Duel.GetTurnPlayer()==tp
+end
+function s.filter(c) 
+	return c:IsAbleToGraveAsCost()
+end
+function s.mtop(e,tp,eg,ep,ev,re,r,rp)
+	local g=Duel.GetMatchingGroup(s.filter,e:GetHandlerPlayer(),LOCATION_HAND,0,nil)
+	if #g>=5 and Duel.SelectYesNo(tp,aux.Stringid(id,1)) then
+		local hg=aux.SelectUnselectGroup(g,e,tp,5,5,nil,1,tp,HINTMSG_TOGRAVE)
```
Comment (larry126) [inline unofficial/c511027120.lua:70]: Don't need SelectUnselectGroup here, it's overkill.
Just a simple g:Select(tp,5,5,nil) should be enough. (and remember the hintselect above)

File: unofficial/c511027120.lua [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+	return (sumpos&POS_FACEDOWN)~=0
+end
+function s.mtcon(e,tp,eg,ep,ev,re,r,rp)
+	return Duel.GetTurnPlayer()==tp
+end
+function s.filter(c) 
+	return c:IsAbleToGraveAsCost()
+end
+function s.mtop(e,tp,eg,ep,ev,re,r,rp)
+	local g=Duel.GetMatchingGroup(s.filter,e:GetHandlerPlayer(),LOCATION_HAND,0,nil)
```
Comment (larry126) [inline unofficial/c511027120.lua:68]: You don't need to use s.filter as the filter is just 1 check. You can do:
```lua
	local g=Duel.GetMatchingGroup(Card.IsAbleToGraveAsCost,tp,LOCATION_HAND,0,nil)
```

File: unofficial/c511027121.lua [hunk 3]
```diff
@@ -0,0 +1,30 @@
+--深夜急行騎士ナイト・エクスプレス・ナイト
+--Night Express Knight (Anime)
+--Scripted by the Razgriz
+local s,id=GetID()
+function s.initial_effect(c)
+	--summon with no tribute
+	local e2=Effect.CreateEffect(c)
```
Comment (larry126) [inline unofficial/c511027121.lua:7]: Not a big issue, but it is preferable to use e1 instead of e2

**Comment by TheRazgriz** [comment]
Changes made.

**Comment by TheRazgriz** [comment]
Fixed indentation.

**Pull request merged**